app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
```

Comment:
1 Tìm hiểu I18n và sử dụng cho toàn bộ các file html nhé

2. dùng `link_to` thay vì `a`